tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {ansible2.2}-{xenial-conf-tests,xenial-mon-osd,xenial-cluster,centos7-mon-osd,centos7-cluster}
+envlist = {ansible2.2}-{xenial-conf-tests,xenial-mon-osd,xenial-cluster,raw-multi-journal,centos7-cluster,dmcrypt-dedicated-journal,dmcrypt-journal-collocation}
```

Comment:
Ok, I've got all these renamed now. I've also made ``centos7-cluster`` and ``xenial-cluster`` use a dedicated journal.